doc/COMPILE.md

```diff
@@ -606,6 +606,27 @@ running cmake like to enable debug symbol.
 For Gentoo it's recommended to emerge sys-apps/lsb-release to name the package
 right even thou not required.
 
+### With Flatpak (Beta)
+
+It is also possible to build a Flatpak package containing Elektra:
+
+```bash
+# install flatpak and flatpak-builder, using the package manager of your distro
+$ apt install flatpak flatpak-builder
+
+# download the runtime and sdk
+$ flatpak install flathub org.kde.Platform//5.15 org.kde.Sdk//5.15
+
+# build and install the package for the current user
+$ flatpak-builder .flatpak-app scripts/flatpak/org.libelektra.kdb.yaml  --force-clean --user --install
+
+# try running the kdb binary
+$ flatpak run org.libelektra.kdb
```

Comment:
Better do something useful like set/get here?

Also clearly warn here, that this is not completely sandboxed and modifies the system (at least user:?).

Reply:
Thanks for the comment! I have added it to the docs.